README.md

```diff
@@ -19,24 +19,39 @@ pip install dscript
 
 ## Usage
 
-Protein sequences need to first be embedded using the Bepler+Berger protein language model; this requires a `.fasta` file as input. Everything before the first space will be used as the key.
+Protein sequences need to first be embedded using the [Bepler+Berger](https://doi.org/10.48550/arXiv.1902.08661) protein language model; this requires a `.fasta` file as input. Everything before the first space will be used as the key.
 
 ```bash
 dscript embed --seqs [sequences] --outfile [embedding file]
+
+#Example
+dscript embed --seqs data/seqs/ecoli.fasta --outfile ecoli_embed.h5
+
 ```
 
 Candidate pairs should be in tab-separated (.tsv) format with no header, and columns for [protein key 1], [protein key 2]. Optionally, a third column with [label] can be provided, so predictions can be made using training or test data files (but the label will not affect the predictions only the first two columns will be read).
 
-While pre-trained model files can be downloaded directly, we recommend instead passing the name of a pre-trained model that will be automatically downloaded from HuggingFace. Available models include:
+While [pre-trained model files](https://d-script.readthedocs.io/en/main/data.html#trained-models) can be downloaded directly, we recommend instead passing the name of a pre-trained model that will be automatically downloaded from HuggingFace. Available models include:
 
 - samsl/dscript_human_v1
-- samsl/topsy_turvy_human_v1 (**recommended**)
+- samsl/topsy_turvy_human_v1 (**recommended**, default)
 - samsl/tt3d_human_v1
 
 ```bash
 dscript predict --pairs [input data] --embeddings [embedding file] --model [model file] --outfile [predictions file]
+
+#Example
+dscript predict --pairs data/pairs/ecoli_toy.tsv --embeddings ecoli_embed.h5 --outfile ecoli_toy_predict
 ```
 
+For inference, proteins can be divided into blocks to reduce memory usage for embeddings using `--blocks`. A GPU to use can be specified with `-d`, followed by the index of a GPU or `all` for all available GPUs.
```

Comment:
Should also specify here how to do cpu inference (or to specify that CPU inference is default)